simulation/deploy_contracts.py

```diff
@@ -45,8 +45,10 @@
         print("Wait for Holder contract to be deployed")
         time.sleep(5)
         continue
-holder_contract_address = holder_receipt['contractAddress']
 
+print(holder_receipt)
+holder_contract_address = holder_receipt['contractAddress']
+print(holder_contract_address)
```

Comment:
This print superfluous, since you print the address of the holder later, and with description.